core/src/com/unciv/ui/pickerscreens/TechButton.kt

```diff
@@ -24,10 +24,10 @@ class TechButton(techName:String, val techManager: TechManager, isWorldScreen: B
         val rightSide = Table()
         val techCost = techManager.costOfTech(techName)
         val remainingTech = techManager.remainingScienceToTech(techName)
-        if (techCost != remainingTech) {
+        if (techCost != remainingTech && isWorldScreen) {
             val percentComplete = (techCost - remainingTech) / techCost.toFloat()
             add(ImageGetter.getProgressBarVertical(2f, 50f, percentComplete, Color.BLUE, Color.WHITE))
```

Comment:
Why remove the progress bar? If someone changed techs halfway, it helps them understand how much of the tech is already completed

Reply:
1.it should be like original game.
2.yesterday i ask you about overflow technology and production. there is a way to play game that research every tech which can be researched to the last turn, then we may get much overflow research point. 
3. tech in every column cost should be different, so it is not useful.

Reply:
1 is not a reason for UI (user experience is more important of gameplay is not affected)
2 what you sent was about a bug in the original, why would I want that?
3 is very useful, it lets you know percentage of completion

Reply:
about 2, i am not sure whether i add this feature.